de_workloads/data_processing/silver/data_factory/adf_pipelines.tf

```diff
@@ -0,0 +1,28 @@
+resource "random_uuid" "deployment" {
+}
+
+resource "azurerm_resource_group_template_deployment" "pipeline_silver" {
+#   count               = var.include_data_quality == false ? 1 : 0
+  name                = "silver-${random_uuid.deployment.result}"
```

Comment:
probably run terraform formatting

Reply:
Auto formatted all the .tf files in VSCode (can perhaps pick up in the linting story if we have specific requirements)

Reply:
@adurkan-amido  if you run -> terraform fmt --recursive  this should format all the tf code in the recursive directory, or you can run terraform fmt  in  individual directories, the TF code still doesn't seem to be formatted

Reply:
It's probably good to mention terraform linting in our linting ADR.

Reply:
`terraform fmt --recursive` isn't picking up any additional formatting issues on these files, let me know if you think there are any specifically (or we address in the linting task as suggested)